book.rb

```diff
@@ -0,0 +1,16 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
```

Comment:
- [ ] Hey there! 😊 Remember that fantastic `many-to-many` relationship between `Person` and `Book`? 📚👥 To achieve the `has-many` association with `Rentals` in the class Book, it's crucial to ensure that the method for adding rentals accepts the book itself as a parameter when creating the new rental. This way, you'll establish a solid link between `Book` and its many `Rentals`, making your project's associations crystal clear! If you have any questions or need assistance with the implementation, don't hesitate to ask. Happy coding! 🚀📖
```suggestion
   def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)

```

Reply:
suggested changes made